src/components/Checkbox/Checkbox.utils.tsx

```diff
@@ -1,4 +1,4 @@
-import { useArgs } from '@storybook/client-api';
+import { useArgs } from '@storybook/preview-api';
```

Comment:
`@storybook/client-api` has changed to `@storybook/preview-api`, so went through and changed these after the calls failed.